presto-cache/src/main/java/com/facebook/presto/cache/alluxio/AlluxioCacheConfig.java

```diff
@@ -87,4 +88,17 @@ public AlluxioCacheConfig setMaxCacheSize(DataSize maxCacheSize)
         this.maxCacheSize = maxCacheSize;
         return this;
     }
+
+    @Config("cache.alluxio.config.validation.enabled")
```

Comment:
Just realized the there are a lot of "dots" in the config names. A dot usually means a subdomain. Use "cache.alluxio.config-validation-enabled". Same for other configs.